build.cake

```diff
@@ -354,6 +354,72 @@ Task("PackageZip")
         Zip(CurrentImageDir, File(ZIP_PACKAGE), zipFiles);
     });
 
+Task("CreateToolManifest")
+    .Does(() =>
+    {
+        var result = StartProcess("dotnet.exe", new ProcessSettings {  Arguments = "new tool-manifest --force" });
+    });
+
+Task("InstallSigningTool")
+    .Description("Installs the signing tool")
+    .IsDependentOn("CreateToolManifest")
+    .Does(() =>
+    {
+        var result = StartProcess("dotnet.exe", new ProcessSettings {  Arguments = "tool install SignClient" });
+    });
```

Comment:
Wonder if https://cakebuild.net/dsl/dotnetcore/ aliases are better than StartProcess?

Reply:
Those aliases are part of the Cake.Incubator project and they don't have ones that support these arguments. There is none for `dotnet new` and I would have to use custom arguments for `DotNetCoreTool` since it is intended to run a tool not install one.

Reply:
Okay, good to know.